src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -0,0 +1,291 @@
+package hex.genmodel.easy;
```

Comment:
This makes sense, but I'm not sure how to recreate your test model jars with my h2o server. Wouldn't I need the native h2o export file rather than the jar? Or did I miss some instructions for recreating the test model in some documentation?